testpar/t_prop.c

```diff
@@ -441,3 +441,192 @@ test_plist_ed(void)
     ret = H5Pclose(acpl);
     VRFY((ret >= 0), "H5Pclose succeeded");
 }
+
+void
+external_links(void)
```

Comment:
Reproducer for the issue from @brtnfld. Aborts with an "invalid communicator" message when the MPI comm on the FAPL used internally for H5Gopen of an external link is invalid (MPI_COMM_NULL in this case since it's the current default comm for FAPLs).